src/Controller/ApiV1Controller.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * Tatoeba Project, free collaborative creation of multilingual corpuses project
+ * Copyright (C) 2009-2010  HO Ngoc Phuong Trang <tranglich@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * PHP version 5
+ *
+ * @category PHP
+ * @package  Tatoeba
+ * @author   HO Ngoc Phuong Trang <tranglich@gmail.com>
+ * @license  Affero General Public License
+ * @link     https://tatoeba.org
+ */
+namespace App\Controller;
+
+use App\Controller\AppController;
+use App\Form\SentencesSearchForm;
+use App\Model\CurrentUser;
+use App\Model\Table\SentencesTable;
+use App\Lib\LanguagesLib;
+use App\Lib\SphinxClient;
+use App\Lib\Licenses;
+use Cake\Core\Configure;
+use Cake\Database\Expression\QueryExpression;
+use Cake\Event\Event;
+use Cake\Routing\Router;
+use Cake\Utility\Hash;
+use Cake\View\ViewBuilder;
+use Exception;
+
+/**
+ * Controller for Api.
+ *
+ * @category Sentences
+ * @package  Controllers
+ * @author   HO Ngoc Phuong Trang <tranglich@gmail.com>
+ * @license  Affero General Public License
+ * @link     https://tatoeba.org
+ */
+class ApiV1Controller extends AppController
+{
+    public $name = 'Sentences';
+    public $components = array (
+        'LanguageDetection',
+        'CommonSentence',
+        'Permissions',
+    );
```

Comment:
It appears you are not using any of these components, so I think it is safe to remove them.

Reply:
OK.

Reply:
Gone:
![image](https://user-images.githubusercontent.com/7943077/117714363-1dbde000-b1d7-11eb-9d86-6fcdb1469628.png)